integrations/gitlab/gitlab_integration/git_integration.py

```diff
@@ -131,7 +143,12 @@ class GitlabPortAppConfig(PortAppConfig):
     project_visibility_filter: str | None = Field(
         alias="projectVisibilityFilter", default=None
     )
-    resources: list[GitlabResourceConfig] = Field(default_factory=list)  # type: ignore
+    filter_bots: bool | None = Field(
+        alias="filterBots",
+        default=False,
+        description="If set to true, bots will be filtered out from the members list. Default value is false",
+    )
```

Comment:
Both Members and Groups depend on this parameter. Removing from top level means I have to include it in integration for groups and members. Please confirm this

Reply:
Also, I placed it at the top level to keep a consistent behavior in this system since groups are related to user. I strictly expect the value of filterBots to be consistent for groups and members. What could go wrong is that a user might specify this parameter as  false for groups kind and true for member kind. Due to the relationship between members and groups, the catalog will be populated with extra inconsistent data.

Reply:
I would add a comment above the `filter_bots` so other developers will understand your motivation.
Also I would rename it to `include_member_bots` and default should be `false`